### DISCUSSION TITLE
Multiple Menu entries for the same dataset

### DISCUSSION THREAD

**Discussion by jose-pr** [zbm-dev/zfsbootmenu#330] [Questions]
Is it possible to create multiple menu entries for the same data but with different org.zfsbootmenu:commandline options?
Example for Pool=boot-pool/ROOT/debian  have two options on the menu one of the which is with console serial and the other one without?
Maybe along the lines of 
org.zfsbootmenu:commandline.0=def_options 
org.zfsbootmenu:commandline.1=def_options console=ttyS0,9600n81

**Comment by zdykstra** [comment]
@ahesford and I are discussing how to best implement this feature, with respect to how ZFSBootMenu works internally. We'll follow up with this discussion once the work starts to take shape and is ready to be merged, so that you can beta test it locally.

**Reply by jose-pr** [reply]
Thanks!

**Reply by jose-pr** [reply]
Any news?

**Reply by zdykstra** [reply]
An interim solution was merged recently that might be useful. You can pass `zbm.kcl_override` in to ZFSBootMenu, and it'll use that for the boot environment kernel command line in place of the `org.zfsbootmenu:commandline` property value.

**Comment by zarMarco** [comment]
I'm add my question. I've installed arch and zfsbootmenu goes. after on same zpool of arch, I've installed void, so now I've root arch on zroot/archroot/default and void on zroot/voidroot/void. But zfsbootmenu see only arch dataset. Can I add dataset of void and dataset in another zpool of freebsd?

**Reply by ahesford** [reply]
ZFSBootMenu can't boot FreeBSD, but you should have no problem finding both Arch and Void. Make sure both `zroot/archroot/default` and `zroot/voidroot/void` have the `canmount=noauto` and `mountpoint=/` properties set and you should see both.

**Reply by zarMarco** [reply]
Thanks so much, this evening I'll try
